CrashReporter.xcodeproj/project.pbxproj

```diff
@@ -3613,6 +3613,7 @@
 				PRODUCT_BUNDLE_IDENTIFIER = "com.microsoft.${PRODUCT_NAME:rfc1034identifier}";
 				PRODUCT_NAME = "Tests iOS";
 				SDKROOT = iphoneos;
+				TARGETED_DEVICE_FAMILY = "1,2";
```

Comment:
Is this change related?

Reply:
This warning (see below) popped up when I fixed unit tests after first commit, so not directly but yes

`TARGETED_DEVICE_FAMILY value does not contain any device family values compatible with the Mac Catalyst platform. Please add the value '2' to the TARGETED_DEVICE_FAMILY build setting. (in target 'Tests iOS' from project 'CrashReporter')`